manifests/params.pp

```diff
@@ -302,7 +382,7 @@
           $service_config_perms = '0600'
         }
         default: {
-          if $majdistrelease <= '5' {
+          if versioncmp($majdistrelease, '5') <= 0 {
```

Comment:
Additionally, `getvar` is in stdlib 2.3.0, so `metadata.json` should not need editing for that either:

https://github.com/puppetlabs/puppetlabs-stdlib/tree/2.3.0/lib/puppet/parser/functions